simulator.py

```diff
@@ -267,7 +267,7 @@ def state(self):
         return [int(self.is_susceptible), int(self.is_exposed), int(self.is_infectious), int(self.is_removed)]
 
     def assert_state_changes(self):
-        next_state = {0:1, 1:2, 2:0}
+        next_state = {0:1, 1:2, 2:3}
```

Comment:
It seems like we're changing a SIS model to a SIR model. Is this intended?

Reply:
It is actually SEIRS. The last two states (RS) transition depend on whether the recovered person transitions to susceptible or is never reinfected/dead (in which case it is just removed).